front/app/components/ConsentManager/Banner.tsx

```diff
@@ -107,10 +107,15 @@ interface Props {
 
 const Banner = ({ onAccept, onChangePreferences, onClose }: Props) => {
   return (
-    <Container tabIndex={0} role="dialog" id="e2e-cookie-banner">
+    <Container
+      tabIndex={0}
+      role="dialog"
+      id="e2e-cookie-banner"
+      aria-labelledby="cookie-banner-title"
```

Comment:
Shall we add a short comment here to explain why we are doing the `aria-labelledby` and link to the MDN docs as well?

(We shouldn't overdo this, but a11y is a bit of an exception because we're not as used to seeing such code as much, and it can help with teaching + remembering.)